src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding fallback font-families in case 'Roboto' fails to load. For example, you could use a generic font family like 'sans-serif' as a backup.